korman/properties/prop_world.py

```diff
@@ -31,7 +31,7 @@ class PlasmaFni(bpy.types.PropertyGroup):
     fog_method = EnumProperty(name="Fog Type",
                               items=[
                                      ("linear", "Linear", "Linear Fog"),
-                                     ("exp2", "Exponential", "Exponential Fog"),
+                                     ("exp", "Exponential", "Exponential Fog"),
```

Comment:
This change *may* impact existing blend files.

Reply:
This was actually tested with an existing blend...in this case, Kotsahvosahn, the waystation. Seemed to export fine and didn't seem to affect it negatively AFAICT.

Reply:
The specific case I'm worried about is if blends that are already set to "exp2" suddenly change to linear or none when opened with this change. Was the blend you tested already set to "exp2" and correctly converted to "exp"?

Reply:
Er, yes...it randomly fogs certain items in linear and exponential stays stuck at its FNI value even with console modification. "None" still seems to do what it's supposed to.

And here I thought this'd be a small change. Ha!

Reply:
So an update...I see why the fog was off...and it wasn't due to the change, but it may be something to address.

The start fog value was larger than the end fog value, which is what was causing the issues above. I switched the values and it works as it should now...although not related, linear fog looks weird for the waystation.

I tried adding a few if/else lines to find this exception and raise an error, but...it crashed Korman and Blender completely. 😮 

Would it be worth raising an error if this happens, and if so, what the best, non-crash way to do so?